### DISCUSSION TITLE
Hyphenation of open source

### DISCUSSION THREAD

**Discussion by bmerry** [astropy/astropy-v5.0-paper#74] [Text comments] [answered]
In some places in the text it is written "open source" (no hyphen) and others it is written "open-source". It always seems to be used as a compound adjective e.g. "open-source software", so I'd suggest it be consistently hyphenated.

**Comment by pllim** [comment]
Hmm, I guess Wikipedia is canon? It has dash there: https://en.wikipedia.org/wiki/Open-source_software

**Comment by eerovaher** [comment, marked as answer]
Fixed by #86.
